src/MaxText/configs/base.yml

```diff
@@ -867,7 +867,9 @@ temperature_tuning: False
 
 # Multimodal flags
 use_multimodal: False
+use_audio: False
```

Comment:
It's a bit confusing that we have both use_multimodal and use_audio flags. Should we change use_multimodel to use_vision? Or is it possible to use one flag use_multimodal for all non-text modalities and depends on the provided inputs to know audio/vision or both? Just thinking out loud. cc @hengtaoguo